tox.ini

```diff
@@ -35,18 +35,6 @@ source =
     flake8_trio
     */site-packages/flake8_trio
 
-[flake8]
```

Comment:
moved so the coverage sections were next to eachother, and removed comment about Y021/Y048